src/static/current-state/components/Lozenge.tsx

```diff
@@ -5,16 +5,18 @@ import { LozengeAppearance } from './types';
 
 let styles = css({
   boxSizing: 'border-box',
-  display: 'inline-flex',
+  display: 'block',
```

Comment:
have you checked the rendering for`inline-block`?

Reply:
All three seem to render the same way.

Reply:
ok cool :)